legend-engine-application-query/src/test/java/org/finos/legend/engine/application/query/api/TestQueryStoreManager.java

```diff
@@ -611,6 +615,12 @@ public void testUpdateQuery() throws Exception
         queryStoreManager.createQuery(TestQueryBuilder.create("1", "query1", currentUser).build(), currentUser);
         queryStoreManager.updateQuery("1", TestQueryBuilder.create("1", "query2", currentUser).build(), currentUser);
         Assert.assertEquals("query2", queryStoreManager.getQuery("1").name);
+        Query queryWithSelectedFields = new Query();
+        queryWithSelectedFields.id = "1";
+        queryWithSelectedFields.versionId = "1.0.0";
+        queryStoreManager.updateQuerySelectedFields("1", queryWithSelectedFields, currentUser);
+        Assert.assertEquals("1.0.0", queryStoreManager.getQuery("1").versionId);
+        Assert.assertEquals("0.0.0", queryStoreManager.getQuery("1").originalVersionId);
```

Comment:
Could we put this in a separate unit test so we're not testing `updateQuery` and `updateQuerySelectedFields` in the same test?